lib/entities/scheduled-action.ts

```diff
@@ -57,6 +57,13 @@ export type ScheduledActionSysProps = {
   updatedBy: Link<'User'> | Link<'AppDefinition'>
 }
 
+export const ScheduledActionReferenceFilters =
+  'sys.contentType.metadata.annotations.ContentType[nin]'
+
+export type ScheduledActionPayloadProps = {
+  withReferences: Record<typeof ScheduledActionReferenceFilters, string[]>
```

Comment:
@luizfonseca WDYT should we mark it optional? I can't think of use case now, but later if there will be other variants of possible metadata, it might be possible that this particular key will not always be present

Reply:
yep, it should be _optional_ otherwise we are implying that whenever payload is present there should be a `withReferences` (not true for _patches_ for example)